modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm

```diff
@@ -444,27 +443,18 @@
 
 //Security Wintercoat (and hood)
 /obj/item/clothing/head/hooded/winterhood/security
+	desc = "A blue, armour-padded winter hood. Definitely not bulletproof, especially not the part where your face goes." //God dammit TG stop putting color in the desc of items like this
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
 	icon_state = "security_wintercoat_hood"
 
-/obj/item/clothing/head/hooded/winterhood/security/hos	//Need to quickly re-define this bc it should still use the winterhood file
-	icon = 'icons/obj/clothing/head/winterhood.dmi'
-	worn_icon = 'icons/mob/clothing/head/winterhood.dmi'
-
 /obj/item/clothing/suit/hooded/wintercoat/security
+	name = "security winter coat" //TG has this as a Jacket now, so unless we update ours, this needs to be re-named as Coat
+	desc = "A blue, armour-padded winter coat. It glitters with a mild ablative coating and a robust air of authority.  The zipper tab is a small <b>\"Armadyne\"</b> logo."
```

Comment:
Only thing worth commenting on: I'm still not quite sure who's logo would be on this - from my understanding, Armadyne supplies armor and weapons, while Lopland supplies clothes and hands; the winter coat is both: armored clothing. So, uh-
Is this something that lore has even solidified? Should I leave it until they do, or just make the decision when sec is all sorted to a Bluesec module?

Doesnt hold the PR though, seems all set; if something breaks, good thing there's like 4 related mirrors that'll follow that we can pin fixes to :,)